manifests/base/jupyter-rocm-minimal-notebook-imagestream.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/rocm"
+    opendatahub.io/notebook-image-name: "ROCm"
+    opendatahub.io/notebook-image-desc: "Jupyter ROCm notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "10"
+    opendatahub.io/recommended-accelerators: '["amd.com/gpu"]'
+  name: jupyter-rocm-minimal
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    # N Version of the image
+    - annotations:
+        opendatahub.io/notebook-software: '[{"name":"ROCm","version":"6.1"},{"name":"Python","version":"v3.9"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.6"},{"name": "Notebook","version": "6.5"}]'
+        openshift.io/imported-from: quay.io/modh/rocm-notebooks
+        opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/notebook-build-commit: $(jupyter-rocm-minimal-notebook-image-commit-n)
+      from:
+        kind: DockerImage
+        name: $(jupyter-rocm-minimal-notebook-image-n)
+      name: "2024.1"
```

Comment:
Since these notebooks will be part of 2024b release this field should change to `2024.2`